src/main-process/atom-application.js

```diff
@@ -1077,7 +1077,7 @@ module.exports = class AtomApplication extends EventEmitter {
 
   initializeAtomHome(configDirPath) {
     if (!fs.existsSync(configDirPath)) {
-      const templateConfigDirPath = fs.resolve(this.resourcePath, 'dot-atom');
+      const templateConfigDirPath = fs.resolve(this.resourcePath, 'templates');
```

Comment:
```suggestion
      const templateConfigDirPath = fs.resolve(this.resourcePath, 'dot-atom');
```
I had renamed dot-atom to templates to make it more concise as to what it was supposed to be, however, that doesn't belong in this PR.

We can revert this line